### ISSUE TITLE
[Question] - Date picker. Stuck. How to render an output?

### ISSUE THREAD

**Issue by breath-ee** [htmlburger/carbon-fields#558]
## Version

  - Carbon Fields: 2.2.0
  - WordPress: 4.9.7
  - PHP: 7.1

## Expected Behavior
Echo output of the date in d-m-Y. 

Although what I really want is to echo only the Y part.

## Actual Behavior
Empty output. I know my code is wrong! 

Also not sure if I've done `set_storage` and `set_input_format` correctly?

## Container definition
```php
Field::make( 'date', 'crb_event_start', 'Event Start' ),
  // ->set_storage_format( 'd-m-Y' ),
  // ->set_input_format( 'd-m-Y', 'd-m-Y' ),
```

```php
<?php echo carbon_get_the_post_meta( 'crb_event_start' ); ?>
```

**Comment by breath-ee** [comment]
I figure it out. Basic example for the documentation.

```php
<?php
  $date = carbon_get_the_post_meta( 'crb_event_start' );
  echo date( 'd-m-Y' );
?>
```

**Comment by deqngeorgiev** [comment]
Hi @breath-ee,

The code from your first comment should work fine. The `->set_storage_format( 'd-m-Y' )` will save the date in `d-m-Y` into the database and then you can show it on the front end with the `<?php echo carbon_get_the_post_meta( 'crb_event_start' ); ?>`

Also, if you want to echo only the `Y` part you can use the [date() function](https://secure.php.net/manual/en/function.date.php)

```
$date = carbon_get_the_post_meta( 'crb_event_start' );
echo date( 'Y', strtotime( $date ) );
```

Regarding the following code from your second comment:

```
<?php
  $date = carbon_get_the_post_meta( 'crb_event_start' );
  echo date( 'd-m-Y' );
?>
```

The `echo date( 'd-m-Y' );` part will print the current date and it has nothing in common with the `crb_event_start` date.

If you have more questions, I will be happy to help.

**Comment by breath-ee** [comment]
@deqngeorgiev, thanks for the reply. 👍 

So I changed the code output back to `<?php echo carbon_get_the_post_meta( 'crb_event_start' ); ?>` and added back `->set_storage_format( 'd-m-Y' )`, now I'm getting an output of _01-07-2018_ which is what I date picked. If commet out `->set_storage_format( 'd-m-Y' )` I get _2018-07-01_. So the code was working as it should, I just can't have been saving the date picker value which is why my output was empty.. doh! 

Also I didn't spot the actual date output of `echo date( 'd-m-Y' );`... thanks.

I've also added back `->set_input_format( 'd-m-Y', 'd-m-Y' )`, which now uses _d/m/y_ from the date picker meta box.

```php
Field::make( 'date', 'crb_event_start', 'Event Start' )
  ->set_storage_format( 'd-m-Y' )
  ->set_input_format( 'd-m-Y', 'd-m-Y' ),
```

Silly errors..
